mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from "clsx";
+import styles from "src/component/text/Text.module.scss";
+
+/**
+ * Type of  text variant styles
+ */
+
+export type ThemeVariant = "clear" | "outline";
+
+/**
+ * Type of size text styles
+ */
+
+export type TextSize = "m" | "l" | "xl";
```

Comment:
We don't need it right now because we don't have design. If we will need it let's add it later

Reply:
delete TextSize